server.go

```diff
@@ -612,17 +612,6 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 			"aux controllers")
 	}
 
-	// For now, the RBF coop close flag and the taproot channel type cannot
-	// be used together.
-	//
-	// TODO(roasbeef): fix
-	if cfg.ProtocolOptions.RbfCoopClose &&
```

Comment:
perhaps we should update the `RbfCoopClose` description just to indicate that if set, it wont be used for tap chans

Reply:
In the sample conf? Sure.